beacon_node/http_api/src/block_id.rs

```diff
@@ -266,6 +267,29 @@ impl BlockId {
             Err(e) => Err(warp_utils::reject::beacon_chain_error(e)),
         }
     }
+
+    pub async fn blob_sidecar_list_indexed<T: BeaconChainTypes>(
+        &self,
+        indices: BlobIndicesQuery,
+        chain: &BeaconChain<T> 
+    ) -> Result<BlobSidecarList<T::EthSpec>, warp::Rejection> {
+        let blob_sidecar_list = self.blob_sidecar_list(&chain).await?;
+        let blob_sidecar_list_indexed = match indices.indices {
+            Some(vec) => {
+                let list = blob_sidecar_list.into_iter()
+                .enumerate()
```

Comment:
The code formatting seems a bit off here and will fail our CI tests, you can fix it by running 

```
cargo fmt --all
```

You may also want to run `clippy` to ensure the changes passes our CI. Some guidelines here in case you haven't seen it:
https://lighthouse-book.sigmaprime.io/contributing.html?highlight=fmt#rust

We'll kick off a CI workflow once this is fixed!